assets/src/edit-story/app/story/useStoryReducer/reducers/combineElements.js

```diff
@@ -76,10 +81,11 @@ function combineElements(state, { firstId, firstElement, secondId }) {
 
   const {
     resource,
-    scale = 100,
-    focalX = 50,
-    focalY = 50,
-    isFill = false,
+    scale = DEFAULT_ATTRIBUTES_FOR_MEDIA.scale,
```

Comment:
Why not just `...DEFAULT_ATTRIBUTES_FOR_MEDIA`?

Reply:
Hm, yes, this could be done cleaner...

Reply:
Changed in dcc1ea85e.

Reply:
Ok. I guess it's ok. Just wanted to clarify. Didn't realize we allowed an empty object for that.